Dockerfile

```diff
@@ -178,4 +182,9 @@ RUN sudo git clone https://github.com/OSGeo/PROJ.git /home/carma/PROJ --branch 6
         
 RUN cd /usr/share/cmake-3.5/Modules && sudo curl -O https://raw.githubusercontent.com/mloskot/cmake-modules/master/modules/FindPROJ4.cmake
 
+# Final system setup this must go last before the ENTRYPOINT
+RUN mkdir -p /opt/carma/routes /opt/carma/logs /opt/carma/launch &&\
```

Comment:
Were these not already defined? Or am I thinking of the CARMA config container only/

Reply:
They were already defined. I just moved this code block to the end and removed some of the folders like app/bin that are no longer used.